portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,25 @@
+/*
+*This class is in charge of each comment object.
+*/
+package com.google.sps.data;
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class Comment  {
```

Comment:
From a design perspective I think it makes more sense to have a Comment contain an individual name, location and comment and to return a list of comments from the server. I don't think it makes sense to change now that this works as is, but just a thought to keep in mind for the future.

My reasoning is that it is a bit less likely to have weird errors introduced as the code evolves in the future where the lists somehow end up out of sync. And it is a bit clearer on the API side that each (name, location, comment) pair goes together in the JSON

Reply:
Hmm yeah that is true. I could have just made an ArrayList of comments. For some reason that did  not cross my mind.